v3/src/utilities/translation/lang/en-US.json5

```diff
@@ -706,8 +706,9 @@
     "DG.AttributeFormat.DatePrecisionShort.millisecond": "M/D/YY HH:mm:ss.SSS",
 
     // DG.AttributeFormulaView
-    "DG.AttrFormView.attrNamePrompt": "Attribute Name:",
-    "DG.AttrFormView.formulaPrompt": "Formula:",
+    "DG.AttrFormView.title": "Edit Formula",
+    "DG.AttrFormView.attrNamePrompt": "name",
+    "DG.AttrFormView.formulaPrompt": "formula",
```

Comment:
I see your point about the inconsistency in the dialogs, but for this PR I think we should stick with the existing strings. I raised the design issue in slack and if there is a decision to change the UI we can address the new/changed strings at that time.

Reply:
Fixed in the last commit.